data/json/overmap/specials.json

```diff
@@ -2236,8 +2234,8 @@
       { "point": [ 1, 1, 0 ], "overmap": "campground_2b_north" }
     ],
     "connections": [ { "point": [ 1, -1, 0 ], "terrain": "road" } ],
-    "locations": [ "wilderness" ],
-    "city_distance": [ 1, 9 ],
+    "locations": [ "forest" ],
```

Comment:
What's the rationale for moving this from the more general 'wilderness' to the more specific 'forest'?

Reply:
Because this didn't look correct.
![campground](https://user-images.githubusercontent.com/44979050/53664857-151e0680-3c6a-11e9-910d-ec4aaf06d32e.png)